scripts/firstboot-windowmanager

```diff
@@ -14,6 +14,12 @@ run_metacity()
     XDG_DATA_DIRS="$new_data_dirs" $1 &
 }
 
+run_mutter()
+{
+    # tell Mutter we want a X11 session, or else it will get confused
+    XDG_SESSION_TYPE="x11" $1 &
```

Comment:
You might want to set up `XDG_DATA_DIRS` as well.

Reply:
I was not sure Mutter supports the same configuration format Metacity does, so I left it out. I actually tried once with the `XDG_DATA_DIRS` but have not seen any difference.

But if you say the format is compatible, I will put it back. :)